src/shared/sleep-config.c

```diff
@@ -108,77 +111,187 @@ int parse_sleep_config(SleepConfig **ret_sleep_config) {
         return 0;
 }
 
+/* Get the list of battery */
+static int battery_enumerator_new(sd_device_enumerator **ret) {
+        _cleanup_(sd_device_enumerator_unrefp) sd_device_enumerator *e = NULL;
+        int r;
+
+        assert(ret);
+
+        r = sd_device_enumerator_new(&e);
+        if (r < 0)
+                return log_oom();
+
+        r = sd_device_enumerator_add_match_subsystem(e, "power_supply", true);
```

Comment:
we nowadays prefer if boolean arguments to function calls come with a comment that explain what argument is passed there, since things are otherwise hard to grok unless checking the function declaraiton. i.e. change to this:

```c
r = sd_device_enumerator_add_match_subsystem(e, "power_supply", /* match= */ true);
```

Reply:
yes. will take care of it.